src/MailCollector.php

```diff
@@ -911,9 +911,12 @@ public function collect($mailgateID, $display = 0)
                         } else {
                             $rejinput['reason'] = NotImportedEmail::MATCH_NO_RULE;
                         }
-                        $refused++;
-                        $rejected->add($rejinput);
-                        $delete[$uid] =  self::REFUSED_FOLDER;
+                        if (!empty($rejinput['to'])) {
```

Comment:
I do not understand why `$rejinput['to']` is empty. Can you explain the usecase ?

Reply:
In fact it's for be sure that the email will be not refused and move in refused folder (but only rejected)

Reply:
Current behaviour is:
 - accepted emails (a ticket or a followup is create) are moved in ACCEPTED_FOLDER,
 - refused emails (email is valid but is refused dur to rules/configuration) are moved into REFUSED_FOLDER,
 - error emails (something goes wrong) are kept in INBOX.

I think we should not change this behaviour.